lib/measured/measurable.rb

```diff
@@ -17,6 +17,8 @@ def initialize(value, unit)
       value
     when Integer
       Rational(value)
+    when String
+      /\d+\/\d+/.match?(value) ? Rational(value) : BigDecimal(value)
```

Comment:
Thought that regex might be slow here
```ruby
n = 5_000_000
COMPILED_REGEX = /\d+\/\d+/

Benchmark.bm do |x|
  x.report { n.times { /\d+\/\d+/.match?("2/1") } }
  x.report { n.times { COMPILED_REGEX.match?("2/1") } }
  x.report { n.times { "2/1".include?("/") } }
end
```
yields
```
    user     system      total        real
0.850214   0.001641   0.851855 (  0.851857)
0.823872   0.001103   0.824975 (  0.824960)
0.485531   0.000751   0.486282 (  0.486314)
```
Regex is ~2x slower, but the amount of initializations have to be really big to see the difference.